config_schema/laser.schema.json

```diff
@@ -17,13 +17,17 @@
                 "type": {
                     "const": "PDM",
                     "description": "Pulse-on-Demand Module from ALPhANOV."
+                },
+                "refresh_interval_ms": {
+                    "type": "integer",
+                    "description": "Refreshing rate, in milliseconds. If not set, there is no auto-update."
+                },
+                "num": {
+                    "type": "integer",
+                    "description": "The index of the PDM device"
                 }
             },
             "$ref": "serial.schema.json",
-            "num": {
-                "type": "integer",
-                "description": "The index of the PDM device"
-            },
             "required": [
                 "num"
```

Comment:
Should this be kept as `num` has been moved in `properties`? (not sure about this)